targets/csharp/templates/PlayFabApiSettings.cs.ejs

```diff
@@ -8,23 +8,31 @@ namespace PlayFab
         public readonly Dictionary<string, string> RequestGetParams = new Dictionary<string, string> {
             { "sdk", PlayFabSettings.SdkVersionString }
         };
-
         /// <summary> This is only for customers running a private cluster.  Generally you shouldn't touch this </summary>
         public string ProductionEnvironmentUrl = PlayFabSettings.DefaultProductionEnvironmentUrl;
         /// <summary> The name of a customer vertical. This is only for customers running a private cluster.  Generally you shouldn't touch this </summary>
         public string VerticalName = <%- getVerticalNameDefault() %>;
+
+#if ENABLE_PLAYFABSERVER_API || ENABLE_PLAYFABADMIN_API
         /// <summary> You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website) </summary>
         public string DeveloperSecretKey = null;
+#endif
+
         /// <summary> You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website) </summary>
         public string TitleId;
+
+#if !DISABLE_PLAYFABCLIENT_API
```

Comment:
Same here.
Please look at the output diff in the repo itself and make sure it looks appropriate.